docs/sources/linking/data-links.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Data links"
+keywords = ["grafana", "data links", "documentation", "playlist"]
+type = "docs"
++++
+
+# Data links
+
+Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization showed four servers, you could add a data link to one or two of them.
+
+Click directly on the panel to see the data link. It appears on the context menu under **Add annotation**.
```

Comment:
`It appears on the context menu under **Add annotation**.`

This is only true for Graph panel. You cannot add annotations o Stat, Gauge, Bar Gauge and Table which also support data links.

Reply:
Then how do you access data links in the other panels?